doc/source/getting_started/installation.rst

```diff
@@ -1,4 +1,57 @@
 Installation
 ------------
 
-*Coming soon!*
\ No newline at end of file
+Conda (Recommended)
+===================
+
+``Drizzlepac`` is installed when you install the ``stenv`` conda environment, formerly (astropy). Select your desired release through the `stenv readthedocs page <https://stenv.readthedocs.io/en/latest/getting_started.html>`_ and follow the instructions for installation. 
+
+
+From Source
+===========
+
+Clone this repository
+*********************
+.. code-block:: shell
+
+    git clone https://github.com/spacetelescope/drizzlepac
+    cd drizzlepac
+
+
+Build the documentation
+=======================
+
+*Note:* If you intend to use ``drizzlepac``'s embedded help feature from within
+an interactive ``python`` or ``ipython`` session, we recommend you do not skip
+this step.
+
+
+.. code-block:: shell
+
+    python setup.py build_sphinx
```

Comment:
calling `setup.py` directly is now deprecated; this will not work. You can try this instead:
```suggestion
    cd doc/
    make html
```